routes/error-tracker.js

```diff
@@ -225,25 +234,28 @@ function getHandler(req, res) {
     },
     severity: severity,
   };
-  const entry = log.entry(metaData, event);
-  log.write(entry, function(err) {
-    if (err) {
-      res.status(statusCodes.INTERNAL_SERVER_ERROR);
-      res.send({error: 'Cannot write to Google Cloud Logging'});
-      res.end();
-      winston.error(appEngineProjectId, 'Cannot write to Google Cloud Logging: '
-        + url.parse(req.url, true).query['v'], err);
-    }
+  unminify.unminify(exception).then(function(unminifiedException) {
+    exception = params.m + '\n' + unminifiedException;
+  }).catch(function(err) {
```

Comment:
Instead of chaining a `catch()`, you can just use the second param to `then()`.
```js
promise.then(onFulfilled, onRejected)
```
In fact, a `catch()` is really just `then(null, callback)`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch